store/store.go

```diff
@@ -194,6 +196,15 @@ func (s *Store) Commit() (root []byte, err lib.ErrorI) {
 	if e := s.writer.Flush(); e != nil {
 		return nil, ErrCommitDB(e)
 	}
+	// Trigger eviction of LSS deleted keys
+	// check if the current version is a multiple of the cleanup block interval
+	if s.Version()%s.config.StoreConfig.CleanupBlockInterval == 0 {
```

Comment:
Should we have a default set if this config is 0?

Reply:
you're right but that should be addressed on the config.go where is being set, will perfom the change there